k256/src/arithmetic/scalar.rs

```diff
@@ -77,6 +85,15 @@ impl Scalar {
         ret[24..32].copy_from_slice(&self.0[0].to_be_bytes());
         ret
     }
+
+    /// Is this scalar greater than or equal to n / 2?
+    pub fn is_high(&self) -> Choice {
```

Comment:
Perhaps this could use a comment similar to the one in `from_bytes` regarding how an overflow results in a borrow?